tests/test_code.py

```diff
@@ -36,9 +36,12 @@ def _get_lintable_files(self, folder):
     def test_linting(self):
         files = self._get_lintable_files('friskby_controlpanel')
         files = files + self._get_lintable_files('tests')
+        files = files + self._get_lintable_files('bin')
```

Comment:
I don't think just adding `bin` to lint folders is enough.  We only collect files ending with `*.py`.  I think it should be enough to also add files starting with `fby_`, i.e., in the walk, add `in fnmatch.filter('fby_*')`.